roles/openshift_prometheus/defaults/main.yaml

```diff
@@ -4,14 +4,13 @@ openshift_prometheus_state: present
 
 openshift_prometheus_namespace: prometheus
 
-openshift_prometheus_replicas: 1
 openshift_prometheus_node_selector: {"region":"infra"}
 
 # images
 openshift_prometheus_image_proxy: "openshift/oauth-proxy:v1.0.0"
-openshift_prometheus_image_prometheus: "openshift/prometheus:v2.0.0-dev"
+openshift_prometheus_image_prometheus: "openshift/prometheus:v2.0.0-dev.3"
```

Comment:
Shouldnt this be 'latest' instead of a specific version?

Reply:
This is what the template uses:
https://github.com/openshift/origin/blob/master/examples/prometheus/prometheus.yaml

Also, I wouldn't want unstable changes in latest to break the deployment